Avoid build failure with -Werror=maybe-uninitialized
--- libipt-2.1/libipt/test/src/ptunit-msec_cache.c.orig	2023-10-12 09:39:11.000000000 +0200
+++ libipt-2.1/libipt/test/src/ptunit-msec_cache.c	2024-01-10 18:37:13.791960352 +0100
@@ -228,7 +228,7 @@ static struct ptunit_result fill_null(vo
 	const struct pt_mapped_section *msec;
 	struct pt_msec_cache mcache;
 	struct pt_image image;
-	struct pt_asid asid;
+	struct pt_asid asid = {0, };
 	int status;
 
 	memset(&mcache, 0, sizeof(mcache));
@@ -300,7 +300,7 @@ static struct ptunit_result read(struct
 static struct ptunit_result fill_nomap(struct test_fixture *tfix)
 {
 	const struct pt_mapped_section *msec;
-	struct pt_asid asid;
+	struct pt_asid asid = {0, };
 	struct pt_section *section;
 	int status;
 
@@ -325,7 +325,7 @@ static struct ptunit_result fill(struct
 {
 	const struct pt_mapped_section *msec;
 	struct pt_section *section;
-	struct pt_asid asid;
+	struct pt_asid asid = {0, };
 	int status;
 
 	memset(&asid, 0, sizeof(asid));
